modules/telnetd/src/main/scala/command/ls.scala

```diff
@@ -0,0 +1,57 @@
+package gem
+package telnetd
+package command
+
+import gem.dao.ProgramDao
+import net.bmjames.opts.{ intOption, help, metavar, short, long, value, strArgument }
```

Comment:
Ben James is the author of `scala-optparse-applicative` which is the API I'm using to define the commands and parsers.